changelog/pending/20240326--sdk-python--the-python-sdk-only-waits-for-pending-outputs-not-all-pending-asyncio-tasks.yaml

```
@@ -0,0 +1,4 @@
+changes:
+- type: fix
+  scope: sdk/python
+  description: The Python SDK only waits for pending outputs, not all pending asyncio tasks
```

```suggestion
  description: Wait only for pending outputs in the Python SDK, not all pending asyncio tasks
```

Sorry for being a pain about these :see_no_evil:

---

It's not a pain